src/Tizen.NUI/src/internal/Interop/Interop.Vector4.cs

```diff
@@ -187,6 +188,9 @@ internal static partial class Vector4
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_delete_Vector4")]
             public static extern void DeleteVector4(global::System.Runtime.InteropServices.HandleRef jarg1);
+
+            [DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Vector4_set_all")]
```

Comment:
I personally prefer to match the usage of InteropServices, but also I understand why you get rid of this. maybe we can update other dllimport also in other time :)